modules/vector-search/pages/pre-filtering-vector-search.adoc

```diff
@@ -0,0 +1,64 @@
+= Pre-filtering Vector Searches
+:page-topic-type: guide
+:page-ui-name: {ui-name}
+:description: You can specify filters as part of a vector search statement which will restrict the documents searched during the query.
+
+[abstract]
+{description}
+
+== About Pre-filtering
+
+Using pre-filtering as part of your vector search offers two key advantages:
+
+[horizontal]
+Enhanced precision and relevance::
+allows searches to be narrowed down to results based on specific criteria (eg., organization, date/time ranges, geospatial locations.)
+
+Performance optimization::
+by reducing the search space before executing the queries, you can improve query execution time and reduce computational overhead.
```

Comment:
I'm not sure that we need to format it like this? The lines are long enough that I just find it pretty hard to read on my screen. 

![image](https://github.com/user-attachments/assets/98356bfe-339a-4b84-9c85-88540e9371b9)
```suggestion

1. **Enhanced precision and relevance**: Narrow your search results based on specific criteria, such as organization, date/time ranges, or geospatial locations.

2. **Performance optimization**: Reduce the search space before executing queries to improve query execution time and reduce computational overhead.
```